vtex/utils/transform.ts

```diff
@@ -347,7 +347,7 @@ export const toProduct = <P extends LegacyProductVTEX | ProductVTEX>(
     kitItems,
   } = sku;
 
-  const videos = isLegacySku(sku) ? sku.Videos : sku.videos;
+  const videos = (sku as LegacySkuVTEX)?.Videos ?? sku.videos;
```

Comment:
shouldn't be better to fix the `isLegacySku` function rather than discard it?

Reply:
fixed it @mcandeia